tests/api/test_retrieve_create_thumbnails.py

```diff
@@ -105,19 +105,6 @@ def test_capture_preview_thumbnail_success(test_app, client, projects):
         assert test_app.fs.get(resp_data['thumbnails']['preview']['storage_id']).__class__ is bytes
 
 
-@pytest.mark.parametrize('projects', [({'file': 'sample_0.mp4', 'duplicate': False},)], indirect=True)
```

Comment:
It's better to adjust this test according to a new logic https://github.com/superdesk/video-server/blob/4ec381561650870dd8aa3375ff705ec4985581d1/src/videoserver/apps/projects/routes.py#L1228